.gitignore

```diff
@@ -13,3 +13,7 @@ data/wuxia.db
 discord.log
 tags
 wuxia_feed/
+data/postgres.json
+README.md
+data/*.log
+data/*.json
```

Comment:
README.md should not be in the gitignore
I go along with the `data/*.log` 
the `data/*.json` also is not entirely correct since there are a few config files in there 

In general I would like this Pull Request to only contain the command changes not this other miscellaneous stuff